brainiak/searchlight/searchlight.py

```diff
@@ -24,6 +25,80 @@
 ]
 
 
+class Shape:
+    """Shape
+
+    Searchlight shape which is contained in a (2*rad+1)^3 cube
+
+    Attributes
+    ----------
+
+    data_ : a 3D boolean numpy array of size (2*rad+1)^3 which is set
+            to True within the boundaries of the desired shape
```

Comment:
As discussed, this is a mask, so it should be named appropriately, e.g., `mask_`.